lib/mongoid/embedded_errors/embedded_in.rb

```diff
@@ -1,12 +1,8 @@
-if Gem::Version.new(Mongoid::VERSION) >= Gem::Version.new('7.0.0')
-  require 'mongoid/association/embedded/embedded_in'
-  ASSOCIATION = Mongoid::Association::Macros::ClassMethods
-  EMBEDS_MANY = Mongoid::Association::Embedded::EmbedsMany
-else
-  require 'mongoid/relations/embedded/in'
-  ASSOCIATION = Mongoid::Relations::Macros::ClassMethods
-  EMBEDS_MANY = Mongoid::Relations::Embedded::Many
-end
+# frozen_string_literal: true
```

Comment:
Let's keep this compatible with both older and new Mongoid. There aren't any benefits in dropping support of the older version.